kong/plugins/standard-webhooks/schema.lua

```diff
@@ -0,0 +1,38 @@
+local typedefs = require "kong.db.schema.typedefs"
+
+local PLUGIN_NAME = "standard-webhooks"
+
+local schema = {
+  name = PLUGIN_NAME,
+  fields = {
+    { consumer = typedefs.no_consumer },
```

Comment:
Since this can't be associated with a consumer should this be allowed to be scoped to a consumer group?

Reply:
I can add the support for this indeed.